### PR TITLE
PVR menu suggestions

### PR DESCRIPTION
**Pull request by hackthis02**
Made the same changes as the master branch. All my changes on the home screen were already in place.

[demo](http://imgur.com/a/Sybdm)

**Recordings menu**:
+Replaced title on sidebar with episode name (_thought this looked
cleaner than having the title on the screen twice_)
+changed date to StartDate (_adds the day the recording was made which
make it easier to determine when the show was aired_)
+changed ActualIcon to Icon on sidebar (_so it displays the show
thumbnail_)
+added genre to sidebar (_didn't know if this was taken out on purpose but I added it back_)

### CODE REVIEW

**Comment by ksooo** [comment]
> +Replaced title on sidebar with episode name (thought this looked
cleaner than having the title on the screen twice

You also removed  episode name from recordings listing. This is okay, but I do not agree to remove title from the sidebar. It would be okay for me add the episode name in addition to the title, though .

**Comment by ksooo** [comment]
> +changed date to StartDate (adds the day the recording was made which

what was displayed there before your change? label2 tells me nothing ;-)

**Comment by hackthis02** [comment]
I'm sorry I thought the title was just the name of the show, which was already being displayed in the list. Is it something else in other addons?

label2 was the date in format MM/dd/yyyy.

**Comment by ksooo** [comment]
> commented 2 minutes ago
I'm sorry I thought the title was just the name of the show, which was already being displayed in the list. Is it something else in other addons?

The sidebar should be self contained. Means it should be understandable without the list on the left side. If you remove the show  title, you remove essential information for the show. No go.

**Comment by hackthis02** [comment]
If that's the case it could be something like 

`''XML
$INFO[ListItem.Title][CS][COLOR button_focus]$INFO[ListItem.EpisodeName][/COLOR] 
''`
on the sidebar?

**Comment by ksooo** [comment]
yep.

**Comment by phil65** [comment]
genre was taken out on purpose btw.

**Comment by hackthis02** [comment]
Just throwing around an idea, What about [this](http://imgur.com/q8OoI1o)
Otherwise this is the the current [idea](http://imgur.com/a/SOOW6)

**Comment by hackthis02** [comment]
Alright I'll remove the genre on my PR.

**Comment by phil65** [comment]
just fyi, what gets displayed for label2 is normally dependent on sortOrder, so it´s not good to use that outside of the list itself.

**Comment by hackthis02** [comment]
I don't believe I did, but that's still something good to know.

**Comment by ksooo** [comment]
> just fyi, what gets displayed for label2 is normally dependent on sortOrder, so it´s not good to use that outside of the list itself.

 so, @hackthis02 fixed a bug by replacing label2 by startdate. nice.

**Comment by ksooo** [comment]
> Just throwing around an idea, What about this

 No way. I don't even understand what is what here.

> Otherwise this is the the current idea

 Okay.

> This is on the master branch, if you like the idea example I'll apply it to the home_alt.

 we should try to keep master and alt_home in sync whenever possible!

**Comment by hackthis02** [comment]
Finished this fix. 

BTW love the direction this branch is going.

**Comment by ksooo** [comment]
please squash the 2 commits down to one. Then, from my pov we are good to go with this PR.

**Comment by ksooo** [comment]
Sorry, not good to go. Just spotted *where* you replaced label2 by startdate. it's in the recordings listing, right? label2 imo has to be kept there. as phil said, the second column of the lsiting shows to value of the current list sort criteria. For consistency reasons this has to be like this everywhee. I was thinking you found label2 in the sidebar where it actually must not appear. So, back to label2, please.

**Comment by hackthis02** [comment]
The issue I saw was when it was label2, if you sorted by anything other then date you got [this](http://imgur.com/a/rej0Y)

**Comment by hackthis02** [comment]
Alright, I'll make the change and squash the commits.

**Comment by hackthis02** [comment]
I believe I've squashed the commits. As I mentioned before I'm new to git.

**Comment by hackthis02** [comment]
I believe I have squashed this one as well.

**Comment by phil65** [comment]
Some quick before-after screenshots would help in order to judge whether this here improves things.

**Comment by hackthis02** [comment]
Sure here is screencaps of both branches before and after. 
http://imgur.com/a/72Jkh

File: 1080i/MyPVRRecordings.xml [hunk 1]
```diff
@@ -173,12 +173,20 @@
...
+						<font>font14</font>
+					</control>
 					<control type="textbox">
-						<top>254</top>
+						<top>282</top>
 						<width>750</width>
 						<height>526</height>
 						<align>justify</align>
 						<label>$INFO[ListItem.Plot]</label>
+						<textcolor>grey</textcolor>
```
Comment (phil65) [inline 1080i/MyPVRRecordings.xml]: what about makin EpisodeName (or whatever is shown above plot) italic? That way we would have a consistent look across different sections, we never use grey font for plot anywhere.

Comment (ksooo) [inline 1080i/MyPVRRecordings.xml]: > what about makin EpisodeName (or whatever is shown above plot) italic? That way we would have a consistent look across different sections, we never use grey font for plot anywhere.

ping @hackthis02

**Comment by hackthis02** [comment]
Sorry guys I was out all weekend. I love that suggestion. I'll make the change and resquash today.

File: 1080i/MyPVRRecordings.xml (hunk 1 above)
Comment (hackthis02) [inline 1080i/MyPVRRecordings.xml]: Sorry guys I was out all weekend. I love that suggestion. I'll make the change and resquash today.

**Comment by hackthis02** [comment]
Okay. edited, submitted, and squashed.

**Comment by phil65** [comment]
Squashed? 5 commits... ;)

**Comment by ksooo** [comment]
> Squashed? 5 commits... ;)

 @hackthis02 please squash down to a single commit. ;-)

**Comment by phil65** [comment]
now itś totally broken. :stuck_out_tongue:

**Comment by hackthis02** [comment]
Sorry guy my squash created conflict. I'll try and figure out what I did wrong. Still getting use to git's command line, more used to TFS or tortoise svn

**Comment by phil65** [comment]
I can recommend SmartGIT (http://www.syntevo.com/smartgit/) in case you prefer to have a GUI. Especially for skinning it is much more convenient.

**Comment by hackthis02** [comment]
Thanks! I'll take a look during lunch.

**Comment by hackthis02** [comment]
Okay, that took longer then I'd care to admit, but I think I got it. Tell me if you see any issues.

**Comment by phil65** [comment]
still broken. As you can see in PR summary, this PR changes 80 files.

**Comment by hackthis02** [comment]
Yeah, things got screwed up when I accidentally sycned to with the main branch. I'll revert back and try committing again.

**Comment by hackthis02** [comment]
Same as other PR.

**Comment by hackthis02** [comment]
Thanks guys for being patient while I learn how to use git.

**Pull request merged**